go.mod

```diff
@@ -1,17 +1,17 @@
 module go.goms.io/fleet
 
-go 1.20
+go 1.21
```

Comment:
Hi Britania! Is this Go + deps version bump intentional?

Reply:
I updated the controller-runtime packages I guess I updated the other as well with it.